test/spec/modules/sovrnBidAdapter_spec.js

```diff
@@ -128,6 +128,42 @@ describe('sovrnBidAdapter', function() {
         expect(impression.video.startdelay).to.equal(startdelay)
       })
 
+      it('sets the proper video object wihtout sizes defined but video sizes defined', function() {
```

Comment:
Added to test if there was no sizes defined that it would use video sizes instead.